services/data/src/links/RestAPILink/fetchData.test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-extra-semi */
```

Comment:
Prettier doesn't approve of the `;()` expression, whereas the tests fail without it.

Reply:
That's obnoxious... i think the typescript prettier plugin solved this previously...

Reply:
Yep, you're right. It's added back in with d56d132.